kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,104 @@ This sample demonstrates how to install the Traefik ingress controller to provid
 load balancing for WebLogic clusters.
 
 ## Install the Traefik operator with a Helm chart
-The Traefik Helm chart is located in the official Helm project `charts` directory at https://github.com/helm/charts/tree/master/stable/traefik.
-The chart is in the default repository for Helm.
+This document is based on Traefik version 2.x with the Helm chart at [Traefik Helm Repository](https://github.com/containous/traefik-helm-chart).
+For more information about Traefik, see the [Traefik Official Site](https://traefik.io/).
 
-To install the Traefik operator in the `traefik` namespace with default settings:
+To install the Traefik operator in the `traefik` namespace with the default settings:
 ```
-$ helm install --name traefik-operator --namespace traefik stable/traefik
+$ helm repo add traefik https://containous.github.io/traefik-helm-chart
+$ helm repo update
+$ kubectl create namespace traefik
+$ helm install traefik-operator traefik/traefik --namespace traefik
 ```
-Or, with a given `values.yaml`:
+You can also install the Traefik operator with a custom `values.yaml`. For more detailed information, see the [Traefik GitHub Project](https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml).
 ```
-$ helm install --name traefik-operator --namespace traefik --values values.yaml stable/traefik
-```
-With the dashboard enabled, you can access the Traefik dashboard with the URL `http://${HOSTNAME}:30305`, with the HTTP host `traefik.example.com`.
```

Comment:
Why were the instructions for accessing the Traefik dashboard, downloading the Traefik helm chart locally, and updating the Traefik values removed? This all seems like useful information.

Reply:
Here we give reference to Official Website for more detail to avoid duplicate information.  There is a sample value.yaml is already there in kubernetes/samples/charts/traefik/vlaues.yaml

Reply:
I don't see links to the doc for these areas - just a top level link to `https://traefik.io/`

Reply:
https://docs.traefik.io/operations/dashboard/